src/api/application-api.md

```diff
@@ -191,7 +191,7 @@ Apart from `el`, you should treat these arguments as read-only and never modify
 
 - **Usage:**
 
-  Mounts a root component of the application instance on the provided DOM element.
+  Mounts a root component of the application instance on the provided DOM element. If application has a `template` option, the `innerHTML` of the DOM element will be replaced with the rendered template.
```

Comment:
```suggestion
  Mounts the root component of the application instance on the provided DOM element. If that root component has a `template` option, the `innerHTML` of the provided DOM element will be replaced with the rendered template.
```

The above seems better, but more generally, I think the "if ..." could cause some miunderstandings:

- If the root component is an SFC, it won't have a template - still the innerHTML will be replaced
- This seems to indicate that I can mount an app without a template. Event though that might technically be possible, it's not very sensical.

We should probably just write

"The `innerHTML` of the provided DOM element will be replaced with the rendered template of the root component." and remove the if clause.

Reply:
I agree with the latter 👍🏻  Changed the description to remove if/else